tools/backups2datalad.py

```diff
@@ -151,6 +153,22 @@ def main(
         di.run(dandisets, exclude)
 
 
+@main.command()
+@click.option("--commitish", metavar="COMMITISH")
+@click.option(
+    "-d",
+    "--dandiset-path",
+    type=click.Path(exists=True, file_okay=False),
+    default=os.curdir,
+)
+@click.argument("dandiset")
+@click.argument("version")
+def release(dandiset, version, dandiset_path, commitish):
+    with DandiAPIClient.for_dandi_instance("dandi") as client:
```

Comment:
let's make it possible to test this against staging!  Those dandisets are smaller and anyone could give it a try/test.

Reply:
Moreover may be it is a good time to establish some rudimentary unittesting for this helper.  Probably based on the dockerized instance as used in dandi-cli so we could programmatically populate some really small dandiset, and do the prototypical workflows of update/releasing and ensure that this datalad'ification script works.  Then it would be `dandi-api-local-docker-tests` instance here (it is ok to expose it straight on CLI without all that DANDI_DEVEL hiding etc)

Reply:
`--dandi-instance` option added.